notebooks/summit/AuxTel/SITCOM-510/SITCOM-510_Linearity_test_using_sky_background.ipynb

```diff
@@ -0,0 +1,715 @@
+{
```

Comment:
<p>you could add cwfs_radius=X if you want to specify it, then pass the <code>radius</code>  argument in the next cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_isotuela/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>Will do. </p>